pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -2009,6 +2009,26 @@ The calculations are done on a sphere.`,
 			}.String(),
 			Volatility: tree.VolatilityImmutable,
 		},
+		tree.Overload{
+			Types: tree.ArgTypes{
+				{"geometry", types.Geometry},
+				{"max_segment_length_meters", types.Float},
```

Comment:
nit: `max_segment_length`
geometry types may not be in metres.

Reply:
removed `meter` both place `max_segment_length_meters` and `info` 
and added extra line `Length units are in units of spatial reference.`